Aptoide/aptoide-client-v8
# [DT-9] Integrate Service Layer - Download + Install

**What does this PR do?**

   Connect the new install manager with the actual stateful install button.

**Database changed?**

   No

**Where should the reviewer start?**

- By commits

**How should this be manually tested?**

  From the bundle with test apps open an app and click install button. Installation state should be the same if navigate back and open again or send app to background and back.

**What are the relevant tickets?**

  Tickets related to this pull-request: [DT-9](https://aptoide.atlassian.net/browse/DT-9)



**Code Review Checklist**
